packages/components/psammead-story-promo-list/src/index.jsx

```diff
@@ -52,6 +40,18 @@ export const StoryPromoLi = styled.li.attrs({
   }
 `;
 
+export const StoryPromoLi = styled(StoryPromoLiThin)`
```

Comment:
Perhaps if `StoryPromoLiThin` is being used as a 'base class', like it is here, it should be called `StoryPromoLiBase`? Initially I thought `StoryPromoLiThin` would have unshared styles, but if that's not the case I don't think it should use a 'variation name'.

This would also be more in-keeping with what's been written in the readme:
> ...`StoryPromoLiThin` ships with no padding presets thereby making it much more configurable to any set of padding/margin rules set on child elements

Reply:
Great idea!